cmake/DaemonFlags.cmake

```diff
@@ -315,7 +315,7 @@ else()
 	endif()
 
 	# Hardening.
-	if (USE_HARDENING OR NOT MINGW)
+	if (USE_HARDENING OR (NOT MINGW AND NOT NACL))
```

Comment:
If that should not be used with NaCl, the `USE_HARDENING` name can be removed from `set(DEFAULT_NACL_VM_INHERITED_OPTIONS …)` in `CMakeLists.txt` (better do both).